src/camvidlog2/yoloe/ai.py

```
@@ -9,14 +10,15 @@
 
 
 # this is a compile time object, so downstream packages can modify it
 # e.g. to support OpenVINO or CUDA and associated dependencies
 class ProvidersList:
     providers = [
-        #    "CUDAExecutionProvider",
-        #    "OpenVINOExecutionProvider",
+        # "CUDAExecutionProvider",
+        # "OpenVINOExecutionProvider",
         "CPUExecutionProvider",
     ]
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)